upup/pkg/fi/cloudup/awstasks/sshkey.go

```diff
@@ -196,6 +196,9 @@ func (e *SSHKey) IsExistingKey() bool {
 }
 
 func (e *SSHKey) TerraformLink() *terraform.Literal {
+	if e.NoSSHKey() {
```

Comment:
I think we'll need this same addition in `RenderCloudformation` a few lines below